virttest/utils_test/__init__.py

```diff
@@ -698,6 +698,29 @@ def run_autotest(vm, session, control_path, timeout,
     from autotest.client.shared.settings import settings
     section_values = settings.get_section_values
 
+    def install_client_tests(url, autotest_path, timeout=600):
+        """
+        Install autotest client tests into local host environment
+        """
+        proxies = {}
+        tmp_file = os.path.join(data_dir.get_tmp_dir(), "client.zip")
+        if params.get("proxy_server"):
+            proxies = dict([params["proxy_server"].split('_proxy:')])
```

Comment:
I think description add in PR https://github.com/autotest/tp-qemu/pull/562 looks better than add it in here.